docs/products/kafka/howto/get-topic-partition-details.rst

```diff
@@ -0,0 +1,46 @@
+Get partition details of a Kafka topic
+==============================================
+
+Use one of three available approaches to retrieve partition details of a Kafka topic.
+
+With the help of Aiven Console
+-------------------------------
+
+In the Aiven Console follow these steps:
+
+1. Log in to the Aiven web console and select your Apache Kafka service.
+2. On the **Topics** tab click the three dots next to the topic you wish to inspect.
+3. In the appeared menu click on the **Info** entry, it will open a modal window with topic details.
+4. Navigate to the **Partitions** tab.
+
+With the help of Aiven API
+---------------------------
+
+Retrieve topic details with an API call using `the endpoint to get Kafka topic info <https://api.aiven.io/doc/#operation/ServiceKafkaTopicGet>`_.
+
+Learn more about API usage in the :doc:`Aiven API overview </docs/tools/api/index>`.
+
+With the help of Aiven CLI
+-------------------------------
+
+Retrieve topic details by using Aiven CLI commands. Find the full list of commands for ``avn service topic`` in :doc:`the CLI reference </docs/tools/cli/service/topic>`.
+
+For example, this bash script, with a help of ``jq`` utility, lists topics and their details for every running Aiven for Apache Kafka service in your project:
+
+.. code:: bash
+
+     #!/bin/bash
+
+     for service in $(avn service list -t kafka --json | jq -r '.[] | select(.state == "RUNNING") | [.service_name,.cloud_name] | join(",")')
```

Comment:
The example feels overly complex, can we just call the `avn service topic-get` without all the for loops around that? I mean, how many times to you need to see all topics in all kafka services you have runnin in a project?

Reply:
Would you introduce some example kafka service name which data we get? However, in this case, a  simple granular  example for `avn service topic-get` we already have in the CLI reference, so it should be something more than that.

Reply:
I agree, but probably I would leave out the outer loop on services. We can loop over topics if needed

Reply:
I believe looping over Kafka services is not needed, we can keep the loop over topics, that still makes sense